cmd/romana.k8s

```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
+
+# Copyright (c) 2016 Pani Networks
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# Usage subcommand.
+usage() {
+	echo "romana: command-line tool for Romana administration"
+	echo "Usage: romana <subcommmand> [options]"
+	echo "       romana <subcommmand> help"
+	echo "       romana [help]"
+	echo "Subcommands:"
+	echo "    add-host"
+	echo "    remove-host"
+	echo "    show-host"
+	echo "    create-owner"
+	echo "    delete-owner"
+	echo "    show-owner"
+	echo "    add-tier"
+	echo "    remove-tier"
+	echo "    add-policy"
```

Comment:
This one only gets installed in a k8s setup. An openstack setup gets its own variant, see https://github.com/romana/romana/tree/kubernetes/cmd .. and it did have openstack-specific bits in it (validating a project name, resolving to an ID and such).

Both are just placeholders though.